plugins/local.py

```diff
@@ -63,6 +64,8 @@ def parse_config(self):
         else:
             raise KeyError("Disabled")
 
+        main["exclude"] = self.get_value("main", "exclude", "")
```

Comment:
In the main configuration, we use the option "excludepks", it is documented as such and the alias "exclude" is there just for compatibility with yum. So, I would propose to either rename this as "excludepkgs", or to support both variants.

Reply:
Is the `exclude` field a true alias of `excludepkgs`? Or are they two separate fields that need to be read independently?

Reply:
Yes, it is: https://github.com/rpm-software-management/libdnf/blob/dnf-4-master/libdnf/conf/ConfigMain.cpp#L470